web/src/Web.App/Controllers/SchoolCustomDataChangeController.cs

```diff
@@ -50,4 +47,38 @@ public async Task<IActionResult> Index(string urn)
             }
         }
     }
+
+    [HttpPost]
+    [Route("financial-data")]
+    [ExportModelState]
+    public IActionResult FinancialData(string urn, [FromForm] SchoolCustomDataViewModel viewModel)
+    {
+        using (logger.BeginScope(new { urn, viewModel }))
+        {
+            try
+            {
+                if (!ModelState.IsValid)
+                {
+                    logger.LogDebug("Posted FinancialData failed validation: {ModelState}",
+                        JsonSerializer.Serialize(ModelState.Where(m => m.Value != null && m.Value.Errors.Any())));
```

Comment:
We have extension method for serialize and deserialize, to apply standard settings and formatting